### PR TITLE
Add errors, retries and retry_rate for cvs output.

### PR DESCRIPTION
**Pull request by charz**

### CODE REVIEW

File: ssbench/reporter.py [hunk 1]
```diff
@@ -169,6 +169,15 @@ def generate_default_report(self, output_csv=False):
...
                                      '%s_count' % label_lc, stats['req_count'])
                     self._add_csv_kv(csv_fields, csv_data,
+                                     '%s_errors' % label_lc,
+                                     stats['errors'])
+                    self._add_csv_kv(csv_fields, csv_data,
+                                     '%s_retries' % label_lc,
+                                     stats['retries'])
+                    self._add_csv_kv(csv_fields, csv_data,
+                                     '%s_retry_rate' % label_lc,
+                                     '%5.2f' % stats['retry_rate'])
```
Comment (dbishop) [inline ssbench/reporter.py]: I think the CSV data should retain full precision (not get rounded to 2 decimals).

Comment (charz) [inline ssbench/reporter.py]: It seems we specify 6 decimals for most value, do you think that's good?

Comment (dbishop) [inline ssbench/reporter.py]: Yeah, I'll buy that :)

**Comment by charz** [comment]
Done

**Comment by timuralp** [comment]
I'm going to squash the commits in this PR and merge it in. Any objections?

**Comment by dbishop** [comment]
sounds good to me.

**Comment by timuralp** [comment]
Closed through https://github.com/swiftstack/ssbench/pull/122

**Pull request closed without merging**